.github/pull_request_template.md

```diff
@@ -57,5 +57,11 @@ If there are user-facing changes then we may require documentation to be updated
 -->
 
 <!--
-If there are any breaking changes to public APIs, please add the `breaking-change` label.
--->
\ No newline at end of file
+If there are any breaking changes to public APIs, please uncomment the following line:
```

Comment:
I have seen projects use markdown check boxes in pr templates for things like these and other |checklist| item like tests etc... might be a good use case here too?

Reply:
I will look into that.